src/classes/wrapped.cpp

```diff
@@ -84,8 +84,7 @@ Wrapped::Wrapped(const StringName p_godot_class) {
 		godot::internal::gdextension_interface_object_set_instance_binding(_owner, godot::internal::token, this, _constructing_class_binding_callbacks);
 		_constructing_class_binding_callbacks = nullptr;
 	} else {
-		ERR_PRINT("BUG: create a Godot Object without binding callbacks.");
-		CRASH_NOW_MSG("BUG: create a Godot Object without binding callbacks.");
+		CRASH_NOW_MSG("BUG: Godot Object created without binding callbacks. Did you forget to use memnew()?");
```

Comment:
Could we include the derived class type that ultimately caused this error?

For example, given the following use case:

```cpp
class MyObject : public RefCounted {
  GDCLASS(MyObject, RefCounted);
  ...
};

class OtherObject : public Control {
  GDCLASS(OtherObject, Control);
  MyObject _object;
};
```
This would trigger this crash because of the instance variable of `MyObject` in `OtherObject`.  If others face this issue, it would be helpful that this error message would indicate that the failure was related to the construction of `MyObject`.  The class type argument to this Wrapped is only `RefCounted`, which is quite meaningless :(

Reply:
It would be really nice to put the name of the derived class in there! Unfortunately, I don't know a way to do it. :-/

Calling `get_class_static()` here will always give us `"Wrapped"`. We could add a virtual `_get_class_namev()` method like we have in Godot itself, but I don't think it'll work in the `Wrapped()` constructor because the virtual table won't have been setup yet, and we'd again just get `"Wrapped"` again.

If someone can figure out how to do this, though, I'd happily include it!

Reply:
Would `_constructing_extension_class_name` not always refer to this class?  When I was fiddling with debugging, I was printing this out and I found it was quite helpful to identify the offender, but maybe I was just simply lucky?

Reply:
Only if `memnew()` is used - it's `memnew()` that calls `_pre_initialize()` which calls `Wrapped::_set_construct_info()` in order to set that.

If `memnew()` is not used, then `_constructing_extension_class_name` will either be `nullptr` or wrong. And, unfortunately, it's the case where `memnew()` is not used that we want to show this message :-)